CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 # Ignite UI for Angular Change Log
 
 All notable changes for each version of this project will be documented in this file.
+
+## 6.2.0
+- `igxGrid`
+    - Initializing an `igxGrid` component without setting height inside a container without height defined now causes the grid's body to 
+    become either 500px high or as high as it is needed to show all records in the current view whichever is smaller.
```

Comment:
This should be in `6.1.3`.

Reply:
Ok, I'll change it and make a PR to 6.1.x then. I didn't consider this a bug and only fixed it for master because of that.